pkgs/applications/misc/obsidian/default.nix

```diff
@@ -52,7 +57,9 @@ let
       mkdir -p $out/bin
       makeWrapper ${electron}/bin/electron $out/bin/obsidian \
         --add-flags $out/share/obsidian/app.asar \
-        --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform=wayland}}"
+        --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform=wayland}}" \
+        --add-flags ${lib.escapeShellArg commandLineArgs} \
```

Comment:
I'm not sure this is needed. I see vscode, chrome, etc. do it, too, however this misses the "single responsibility" concept. Wouldn't potentially all packages benefit from such an abstraction?

Reply:
I forgot to mention the reason behind my comment: you could always wrap or override the derivation in your own code to achieve it.

Reply:
Also found this kinda ugly. A global chromium argument setting would be nice, but for now I'll just look into creating my own wrapper and then I'll remove the commit.

Reply:
Found an easy solution in <https://nixos.wiki/wiki/Nix_Cookbook>. Solution is in the updated merge request and removed the commit.

Reply:
Nice!